guides/inference-scheduling/ms-inference-scheduling/values.yaml

```diff
@@ -13,15 +13,6 @@ routing:
     connector: nixlv2
     secure: false
 
-  inferencePool:
-    create: false
-
-  httpRoute:
-    create: false
-
-  epp:
-    create: false
-
```

Comment:
PR description does not mention why this is removed (here and in helmfile template), is it v0.3 change?